docs/contributors/write-documentation.rst

```diff
@@ -0,0 +1,185 @@
+    Refer to `Google's docstring guide <https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings>`_ for that, or
+    checkout full-fleshed examples in `Numpy style <https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy>`_ or `Google style <https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google>`_. We also have a short intro in the :doc:`next section </contributors/write-docstrings>`.
+
+Learn by example
+^^^^^^^^^^^^^^^^
+
+The next section will introduce many of the cool features of ``.rst`` + Sphinx + plugins.
+
+Click on "*Code for the example*" to look at the ``.rst`` code that generated what you are reading.
+
+.. tab-set::
+
+    .. tab-item:: Full-fleshed ``.rst`` example
+
+        .. include:: example.rst
+
+    .. tab-item:: Code for the example
+
+        .. literalinclude:: example.rst
+            :language: rst
+
+FAQ
+---
+
+.. dropdown:: How do I create new manual documentation files.
+
+    - Create a new ``.rst`` file in the ``docs/`` folder
+    - List it in ``docs/index.rst`` file under the ``.. toctree::`` directive
+    - **Or** create a subfolder in ``docs/`` with an ``index.rst`` file.
+        - This is useful for grouping documentation files together.
+        - ``docs/{your_subfolder}/index.rst`` should contain a ``.. toctree::`` directive listing the files in the subfolder.
+        - It should also be listed in the ``docs/index.rst`` under the ``.. toctree::`` directive to appear on the left handside of the documentation.
+
+    You can look at the |contributors|_ folder for an example.
+
+.. dropdown:: How do I document a sub-package like :py:mod:`gflownet.proxy.crystals`?
+
+    Just add a docstring at the top of the ``__init__.py`` file of the sub-package:
+
+    .. code-block:: python
+
+        """
+        This is the docstring of the sub-package.
+
+        It can contain any kind of ``.rst`` syntax.
+
+        And refer to its members: :meth:`~gflownet.proxy.crystals.crystal.Stage`
+
+        .. note::
+            This is a note admonition.
+
+        """
+
+    You can similarly document a **module** by adding a docstring at the top of the file
+
+.. dropdown:: How do I document a module varaible?
+
+    Add a docstring **below** the variable to document like
+
+    .. code-block:: python
+
+        MY_VARIABLE = 42
+        """
+        This is the docstring of the variable.
+
+        Again, It can contain any kind of ``.rst`` syntax.
+        """
+
+.. dropdown:: How do I document a class?
+
+    Currently, ``autoapi`` is setup to consider the documention of a class to be the same as the documentation for the ``__init__`` method of the class.
+
+    This can be changed by changing the ``autoapi_python_class_content = "init"`` configuration variable in ``docs/conf.py``. See `AutoAPI <https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html#confval-autoapi_python_class_content>`_ for more details.
```

Comment:
picky suggestion :P
```suggestion
    This can be modified by changing the ``autoapi_python_class_content = "init"`` configuration variable in ``docs/conf.py``. See `AutoAPI <https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html#confval-autoapi_python_class_content>`_ for more details.
```